src/services/export.ts

```diff
@@ -17,8 +17,8 @@ interface DatasetRow {
   group?: string
 }
 
-export const exportDatasets = async (
-  datasets: DatasetProperties[]
+export const exportFactors = async (
+  elements: ResolvedFactor[]
```

Comment:
Same thread as https://github.com/opendatateam/udata-front-kit/pull/756/files#r2324682523.
Given [how it's used below](https://github.com/opendatateam/udata-front-kit/pull/756/files#diff-8948949693f80431424790a7c105b6f6423db8d25dcd721d7170e01d4c571d48R36), I'm not sure we can expect anything else than factors in `elements`.